lib/easymon/checks/multi_active_record_check.rb

```diff
@@ -0,0 +1,52 @@
+module Easymon
+  class MultiActiveRecordCheck
+    attr_accessor :block
+    attr_accessor :results
+
+    # Here we pass a block so we get a fresh instance of ActiveRecord::Base or
+    # whatever other class we might be using to make database connections
+    #
+    # For example, given the following other class:
+    # module Easymon
+    #   class PrimaryReplica < ActiveRecord::Base
+    #     establish_connection :"primary_replica"
+    #   end
+    #
+    #   class OtherReplica < ActiveRecord::Base
+    #     establish_connection :"other_replica"
+    #   end
+    # end
+    #
+    # We would check both it and ActiveRecord::Base like so:
+    # check = Easymon::MultiActiveRecordCheck.new {
+    #    {
+    #      "Primary": ActiveRecord::Base.connection,
+    #      "PrimaryReplica": Easymon::PrimaryReplica.connection
+    #      "OtherReplica": Easymon::OtherReplica.connection
+    #    }
+    # }
+    # Easymon::Repository.add("multi-database", check)
+
+    def initialize(&block)
+      self.block = block
+    end
+
+    def check
+      connections = Hash(@block.call)
+
+      results = connections.transform_values {|connection| database_up?(connection)}
```

Comment:
Doesn't quite fit our house style, perhaps we could set up our rubocop-37signals gem? We can follow local_time:

1. https://github.com/basecamp/local_time/blob/main/Gemfile#L6
2. https://github.com/basecamp/local_time/blob/main/.rubocop.yml

Reply:
Fixed in https://github.com/basecamp/easymon/pull/43